api/serializers.py

```diff
@@ -0,0 +1,108 @@
+from rest_framework import serializers
+from rest_framework.pagination import PageNumberPagination
+
+from game.models import Game, Genre, Platform, Screenshot, Cover, UserFavouriteGame
+from django.conf import settings
+
+
+class MainPageGamesSetPagination(PageNumberPagination):
+    """Sets games limit per page displayed on Main Page"""
+
+    page_size = settings.GAMES_PER_PAGE_MAIN
+
+
+class MustsPageGamesSetPagination(PageNumberPagination):
+    """Sets games limit per page displayed on Musts Page"""
+
+    page_size = settings.GAMES_PER_PAGE_MUSTS
+
+
+class GameSerializer(serializers.ModelSerializer):
+    """Serializer for Game Model"""
+
+    class Meta:
+        model = Game
+        fields = '__all__'
+
+    def to_representation(self, instance):
+        """Ensures only not null keys are displayed"""
+
+        result = super(GameSerializer, self).to_representation(instance)
```

Comment:
This is old-style usage of super, in python3 just

`result = super(GameSerializer, self).to_representation(instance)`